extensions/podman/package.json

```diff
@@ -269,6 +277,41 @@
             ]
           ]
         },
+        {
+          "id": "checkUnsupportedPodmanMachineCommand",
+          "title": "Checking if an old unsupported Podman machine exists",
+          "when": "!isLinux",
```

Comment:
Should this be isMac?

Reply:
so I could but there is another issue where in this checkUnspportedPodmanMachineCommand where we will check v4 --> v5 machines like applehv --> applehv or on Windows so I'll just enhance the method